newsroom/topics/views.py

```diff
@@ -58,8 +59,11 @@ async def post_topic(args: RouteArguments, params: None, request: Request):
             "user": user_dict.get("_id"),
             "company": user_dict.get("company"),
             "_id": ObjectId(),
-            "created_filter": topic.pop("created", None),
             "is_global": topic.get("is_global", False),
+            # `_created` needs to be set otherwise there is a clash given `TopicResourceModel` and
```

Comment:
When constructing a model from a dict, you use the aliased name for fields.
In this case we have the [created](https://github.com/devketanpro/newsroom-core/blob/NHUB-531/newsroom/topics/topics_async.py#L48) field, which is used to provide a filter for [creation date](https://github.com/devketanpro/newsroom-core/blob/NHUB-531/newsroom/topics/views.py#L191).

So here, `_created` and `created` have 2 different uses, and only `created` should need to be populated as `_created` is populated by the async Newshub resource service

Reply:
Yeah, that's what I initially thought but when setting `created` attribute only (for the creation date filter) it throws a validation error for the `created` field that holds the date time instead. So there is some sort of clash between aliases and members here. 

<img width="904" alt="image" src="https://github.com/user-attachments/assets/cd88ca61-00df-4290-81d7-a3f3f7038546">